simplq/src/components/common/Nav/LeftNav.jsx

```diff
@@ -3,34 +3,34 @@
 import React from 'react';
 import styles from './nav.module.scss';
 import { smoothScrollTo } from '../utilFns';
-
 import LoginButton from '../LoginButton';
 
-const LeftNav = ({ open, toggleClose, page }) => {
+const LeftNav = ({ open, toggleClose }) => {
   const scrollToHowItWorks = () => {
     // Close the navbar on click
     toggleClose();
     // get the target div by ID
     const element = document.getElementById('target_how_it_works');
-    smoothScrollTo(element);
+    if (element) {
+      // element is on the current page, just have to scroll to it
+      smoothScrollTo(element);
+    } else {
+      window.location.href = '/#target_how_it_works';
+    }
```

Comment:
Clicking How It Works on another page redirects to the homepage, but doesn't scroll to the exact 'how it works' section, but instead, to the Why SimplQ section.

Reply:
Oh is it, I tried here on chrome and it's working. Can you help me reproduce the issue?

Reply:
https://user-images.githubusercontent.com/12583528/103136152-ecd78580-46e3-11eb-9b69-a84f8ab1a612.mp4


This is how it usually goes for me on Ubuntu with firefox. But I noticed it worked a few times intermittently.

Reply:
I was able to reproduce this. It seems like a bug with firefox, I've resolved it.

https://github.com/SimplQ/simplQ-frontend/pull/441/commits/2e09215613803c16d16c831edbe4ca524166301b

Reply:
Cool, looks like it works now.